dev/com.ibm.ws.jmx.connector.client.rest/src/com/ibm/ws/jmx/connector/client/rest/internal/RESTMBeanServerConnection.java

```diff
@@ -2377,6 +2378,13 @@ public void run() {
                                     }
                                 } else {
                                     //no-op for failover polling, just break into the sleep segment
+                                    // Server sends a string back that needs to be consumed to close the connection
+                                    InputStream is = connection.getInputStream();
+                                    int data = is.read();
+                                    while (data != -1) {
```

Comment:
Just want to make sure that it won't get stuck here indefinitely. Otherwise, a maximum retry could be set.